app/helpers/session_helper.rb

```diff
@@ -5,7 +5,7 @@ def show_session(question)
   end
 
   def show_link(step)
-    "<a href='#{new_registration_path(step)}'>Change</a>".html_safe
+    "<a href='#{new_registration_path(step)}' style='color:#551A8B'>Change</a>".html_safe
```

Comment:
Can we give this a class instead of hardcoding a style?

Reply:
Sure, no probs, is a temp fix for an odd issue, the text color for the telephone link on the equivalent/overseas completion page was different to the others, not sure why yet.

Reply:
May be worth flagging, could just be an oversight